attestation-agent/attester/src/lib.rs

```diff
@@ -77,12 +77,14 @@ pub enum InitDataResult {
     Unsupported,
 }
 
+pub(crate) type TeeEvidence = serde_json::Value;
```

Comment:
I love this definition and change.

Reply:
I personally don't love it so much, since this is just an alias to serialization library, and, if possible, we should not expose this in a the API (this leaves the option to later switch it to sth else, for e.g. more resource-constrained TEEs). We can maybe use a Newtype (`struct TeeEvidence(serde_json::Value)` the evidence.0 shouldn't be pub, so we don't leak imoplementation details)?

Reply:
it could be dropped from this series if it's not strictly necessary?

Reply:
I'm not too worried about requiring attesters to use serde_json. Every single one does so already. The case of a resource constrained TEE seems out of scope since that will probably require changing a bunch of other stuff in guest components (i.e. all the other places we use serde_json).